core/tests/ts-integration/src/env.ts

```diff
@@ -45,25 +42,142 @@ export async function waitForServer() {
     throw new Error('Failed to wait for the server to start');
 }
 
+function getMainWalletPk(network: string): string {
+    if (network.toLowerCase() == 'localhost') {
+        const testConfigPath = path.join(process.env.ZKSYNC_HOME!, `etc/test_config/constant`);
+        const ethTestConfig = JSON.parse(fs.readFileSync(`${testConfigPath}/eth.json`, { encoding: 'utf-8' }));
+        return ethers.Wallet.fromMnemonic(ethTestConfig.test_mnemonic as string, "m/44'/60'/0'/0/0").privateKey;
+    } else {
+        return ensureVariable(process.env.MASTER_WALLET_PK, 'Main wallet private key');
+    }
+}
+
+async function loadTestEnvironmentFromFile(chain: string): Promise<TestEnvironment> {
```

Comment:
This method has a bunch of similarities with  `loadTestEnvironmentFromEnv`. But soon we will remove `loadTestEnvironmentFromEnv` and keep the only option